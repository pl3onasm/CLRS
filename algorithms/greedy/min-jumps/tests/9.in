2 5 1 3 8 1 5 0 7 7 0 1 2 5 4 1 8 2 1 0 0 2 0 1 2 3 4 1 0 1 1 4 2 3 8 0 0 8 5 1 1 6 3 5 0 2 3 2 1 2 4 3 1 2 6 9 4 3 0 1 0 2 0