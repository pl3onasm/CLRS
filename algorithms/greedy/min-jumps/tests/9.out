Min jumps: 16
Path:
  0, 1, 4, 9, 16, 24, 26, 30, 31, 34, 41, 46, 49, 50, 53, 55, 62
